Automattic/crowdsignal-ui
# Add Dropdown block

## Summary

This PR adds the Dropdown block for the new editor.

Depends on: D82342-code

Ref: c/5G7VBwCE-tr

## Test Instructions

* Make sure to apply the backend patch
* Create a new project and use one of the inserters to include the Dropdown block
* Play with the block to include the desired options. It should support pasting a list of options
* Go to the renderer and check if the block works as a regular dropdown component
* It should support keyboard navigation

## Screenshots
![image](https://user-images.githubusercontent.com/7811225/173134758-bab5c33c-0767-4112-a78c-942c084d8e3c.png)

![image](https://user-images.githubusercontent.com/7811225/173134676-85ce4cf0-84c5-4379-8469-00fc66dd8f76.png)
